submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/request/SubmissionRequestLoader.kt

```diff
@@ -14,16 +15,21 @@ import uk.ac.ebi.extended.serialization.service.FileProcessingService
 private val logger = KotlinLogging.logger {}
 
 class SubmissionRequestLoader(
-    private val submissionPersistenceQueryService: SubmissionPersistenceQueryService,
+    private val queryService: SubmissionPersistenceQueryService,
+    private val persistenceService: SubmissionPersistenceService,
     private val fileProcessingService: FileProcessingService,
 ) {
 
-    internal fun loadRequest(accNo: String, version: Int): ProcessedSubmissionRequest {
+    /**
+     * Calculate md5 and size for every file in submission request.
+     */
+    internal fun loadRequest(accNo: String, version: Int): ExtSubmission {
         logger.info { "Started loading request accNo='$accNo', version='$version'" }
-        val original = submissionPersistenceQueryService.getPendingRequest(accNo, version)
+        val original = queryService.getPendingRequest(accNo, version)
```

Comment:
I think we should include additional logging in this method since, right now, we don't have any feedback about what is happening while loading the request and it might take a while for some submissions